lib/etcher.js

```diff
@@ -56,6 +91,11 @@ electron.app.on('ready', function() {
         mainWindow.show();
       }, 100);
 
+      if (args._.length === 1) {
+        mainWindow.webContents.executeJavaScript(
+          'angular.element(document.getElementById("appcontainer")).scope().app.selectImage("' + args._[0] + '");'
```

Comment:
This looks OK, however I'd like to find another way to do this rather than relying on evaluating a string as JavaScript, and knowing too much about the controller's API from outside the Angular's world. I'm thinking about something like this:

- Have a new file under `src/`, which runs on the main process, to store the CLI options state.
- From the renderer process, remotely require this file (with `electron.remote`) when the Angular application starts to get the values. For modularisation reasons, creating a new Angular service to encapsulate the details of fetching the appropriate remote file would be great.
- Call the appropriate functions (like `selectImage()`) from the Angular's world, if certain options exist.